rest_framework_filters/filterset.py

```diff
@@ -182,7 +182,7 @@ def get_param_filter_name(cls, param, rel=None):
             return param
 
         # Attempt to match against exclusion filters
-        if param[-1] == '!' and param[:-1] in cls.base_filters:
+        if param and param[-1] == '!' and param[:-1] in cls.base_filters:
```

Comment:
I wouldn't add the check here. If there's no param name, the entire function can be skipped. I would add the following to the very start of the function:

```python
# Skip empty parameter names.
if not param:
    return param
```

Reply:
Updated!